index.ftd

```diff
@@ -1,51 +1,1944 @@
+color: $inherited.colors.text-strong
+role: $inherited.types.heading-small
+
+-- end: ftd.row
+
+-- fallback-title:
+if: { logo-with-name.site-logo == NULL }
+site-name: $logo-with-name.site-name
+
+-- end: ftd.row
+
+-- end: logo-with-name
+
+
+
+
+
+
+
+
+
+
+-- component fallback-title:
+optional string site-name:
+
+-- ftd.text: LOGO
+if: { fallback-title.site-name == NULL }
+role: $inherited.types.heading-large
+color: $inherited.colors.text-strong
+
+-- end: fallback-title
+
+
+
+
+
+
+
+
+
+
+-- component footer:
+optional string site-name:
+optional string site-url:
+optional ftd.image-src logo-1: $assets.files.assets.meta.png
+optional ftd.image-src logo-2: $assets.files.assets.figma.png
+optional ftd.image-src logo-3: $assets.files.assets.google.png
+optional ftd.image-src logo-4: $assets.files.assets.messanger.png
+optional ftd.image-src logo-5: $assets.files.assets.basketball.png
+optional string twitter-url:
+optional string facebook-url:
+optional string discord-url:
+optional string instagram-url:
+optional string telegram-url:
+optional string copyright: Copyright © 2023 - fifthtry.com
+optional body bio:
+integer logo-width: $common.logo-width
+integer logo-height: $common.logo-height
+pr.toc-item list footer-links: $common.footer-links
+common.social-media list social-links: $common.social-links
+optional ftd.image-src site-logo: $assets.files.assets.google.svg
+integer logo-height: 32
+boolean social: false
+optional ftd.color background-color: $inherited.colors.background.step-1
+pr.toc-item list footer-list: $common.sitemap.sections
+
+-- ftd.column:
+width: fill-container
+
+-- ftd.desktop:
+
+-- footer-desktop:
+site-name: $footer.site-name
+telegram-url: $footer.telegram-url
+instagram-url: $footer.instagram-url
+twitter-url: $footer.twitter-url
+discord-url: $footer.discord-url
+facebook-url: $footer.facebook-url
+logo-5: $footer.logo-5
+logo-4: $footer.logo-4
+bio: $footer.bio
+copyright: $footer.copyright
+logo-height: $footer.logo-height
+background-color: $footer.background-color
+footer-list: $footer.footer-list
+logo-1: $footer.logo-1
+logo-2: $footer.logo-2
+logo-3: $footer.logo-3
+site-url: $footer.site-url
+site-logo: $footer.site-logo
+social: $footer.social
+
+-- end: ftd.desktop
+
+-- ftd.mobile:
+
+-- footer-mobile:
+site-name: $footer.site-name
+telegram-url: $footer.telegram-url
+instagram-url: $footer.instagram-url
+twitter-url: $footer.twitter-url
+discord-url: $footer.discord-url
+facebook-url: $footer.facebook-url
+logo-5: $footer.logo-5
+logo-4: $footer.logo-4
+bio: $footer.bio
+copyright: $footer.copyright
+logo-height: $footer.logo-height
+background-color: $footer.background-color
+footer-list: $footer.footer-list
+logo-1: $footer.logo-1
+logo-2: $footer.logo-2
+logo-3: $footer.logo-3
+site-url: $footer.site-url
+site-logo: $footer.site-logo
+social: $footer.social
+
+-- end: ftd.mobile
+
+-- end: ftd.column
+
+-- end: footer
+
+
+
+
+
+
+
+
+
+
+-- component footer-desktop:
```

Comment:
@saurabh-lohiya footer component should be like:
https://github.com/fastn-community/set-1-ds/blob/8c2335a217c3a4fa37f829cd4280307baccf51ee/index.ftd#L576